src/content/en/tools/workbox/reference-docs/_redirects.yaml

```diff
@@ -38,3 +38,6 @@ redirects:
 
 - from: /web/tools/workbox/reference-docs/v4.0.0/...
   to: /web/tools/workbox/reference-docs/v4.2.0/...
+
+- from: /web/tools/workbox/reference-docs/v4.2.0/...
```

Comment:
Now there's going to be a double-redirect for anyone who goes to `v4.0.0`, where it takes them to `v4.2.0`, and then where that takes them to `latest`.

What I've been doing with these redirects is not to use the `latest` alias at all, and just explicitly update any `vX.Y.Z` URLs to the explicit latest version in the `vX` major release.

Reply:
The problem in this case is there's no `v4.3.0` directory.